lambda/custom/document/renderPage.json

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
This document does not render in the authoring tool without errors. https://developer.amazon.com/alexa/console/ask/displays

Reply:
easy fix